src/parser/core/modules/Suggestions/Suggestion.ts

```diff
@@ -0,0 +1,78 @@
+import {ReactNode} from 'react'
+import {matchClosestLower} from 'utilities'
+
+export const SEVERITY = {
+	// NOTE: Don't use MORBID, It's for deaths. I _will_ block the PRs on this.
+	MORBID: 0,
+	MAJOR: 1,
+	MEDIUM: 2,
+	MINOR: 3,
+	// The matchClosest fall back to undefined, so let's use that for ignore too
+	IGNORE: undefined,
+}
+
+export interface SuggestionOptions {
+	icon: string
+	content: ReactNode
+	why: ReactNode
+	severity?: number
+}
+
+export default class Suggestion {
+	public icon: string // TODO: default image
+	public content: ReactNode
+	public why: ReactNode
+	private severityValue?: number
```

Comment:
The undefined option is basically there because of how TieredSuggestion passes things into the constructor.  However, it doesn't really need to be there for the base class field otherwise, since non-tiered Suggestions should always be passing one in.  Should be able to tweak these around a bit to get rid of it here.